patches/runtime/0010-Remove-duplicate-property.patch

```diff
@@ -0,0 +1,31 @@
+From af031b51511a210b4a496107a1229ae3a4598c46 Mon Sep 17 00:00:00 2001
+From: Chris Rummel <crummel@microsoft.com>
+Date: Tue, 1 Sep 2020 15:44:55 -0500
+Subject: [PATCH 10/10] Remove duplicate property
```

Comment:
Not clear what the duplicate property is here, what problem this causes. Somewhere else in the file?

Reply:
Added a comment in the patch.